bundles/org.openhab.binding.elroconnects/README.md

```diff
@@ -0,0 +1,190 @@
+# ElroConnects Binding
+
+The ElroConnects binding provides integration with the [Elro Connects](https://www.elro.eu/en/smart-home) smart home system.
```

Comment:
I believe from looking at their website that the correct way to write their name is `ELRO Connects` note the capitals. Please change any reference to their name to respect how they wish their brand to be presented in the readme. This matters to some companies.